src/main/java/hwr/oop/exercises/tictactoe/TicTacToe.java

```diff
@@ -1,27 +1,118 @@
 package hwr.oop.exercises.tictactoe;
 
+import java.util.Arrays;
+
 class TicTacToe {
 
-    TicTacToe() {
-        // TODO implement!
+    static int[][] board;
```

Comment:
Warum `static`? In der Vorlesung kommen wir später noch auf `static` (Klassenmethoden und Klassenattribute). Soviel vorweg: Man sollte es wo möglich _vermeiden_ und insb. non-final `static` Attribute sind eher ein Antipattern.